### DISCUSSION TITLE
Checkout App Channels not visible

### DISCUSSION THREAD

**Discussion by palash-cj** [saleor/saleor#12043] [General]
Hi!
I want to implement the settings of payment provider through saleor dashboard, for this channels within the checkout app should be selected. I installed the Checkout through manifest url. But channels are not visible as the graphql api for operation:"Channels" has requestURL as "https://localhost:8000". I checked the whole saleor folder but didn't find such url where localhost:8000 is "https". Please help me fetch channels from this graphql api.

Please help me proceed to payment provider settings.

**Comment by peelar** [comment]
Hello @palash-cj 👋 

It appears you didn't set the env vars. You can find instructions on it [here](https://github.com/saleor/react-storefront/blob/canary/docs/development.md).

**Comment by palash-cj** [comment]
Hi @peelar 
Thanks for the answer.

But the channels were not visible in dashboard because it was requesting a localhost, but it needs https client to proceed for checkout . As soon as I changed that API_URL to https client, it worked and channels are visible.
